readthedocs/core/utils/__init__.py

```diff
@@ -95,7 +95,12 @@ def prepare_build(
         return (None, None)
 
     if not version:
-        version = project.versions.get(slug=LATEST)
+        default_branch = project.default_branch
```

Comment:
Ok, I got confused, we need default_version here, because default_branch is used like

>  What branch "latest" points to. Leave empty to use the default value for your VCS (eg. trunk or master).

Default version only list versions that are active

And I think we can use the original method mentioned in https://github.com/rtfd/readthedocs.org/pull/5393#discussion_r262113965

Reply:
@stsewd 
Ok.. I have pushed the changes.